test/Sentry.Analyzers.Tests/Sentry.Analyzers.Tests.csproj

```diff
@@ -1,22 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <TargetFramework>$(PreviousTfm)</TargetFramework>
-        <Nullable>enable</Nullable>
+  <PropertyGroup>
+    <TargetFramework>$(PreviousTfm)</TargetFramework>
+    <Nullable>enable</Nullable>
 
-        <IsPackable>false</IsPackable>
-    </PropertyGroup>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
 
-    <ItemGroup>
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Analyzer.Testing" Version="1.1.2"/>
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeFix.Testing" Version="1.1.2"/>
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.3.0"/>
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.3.0"/>
-    </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Analyzer.Testing" Version="1.1.3-beta1.25564.1" />
```

Comment:
`beta`??? Should we stick to the stable packages?

Reply:
NU1510 usually means the package reference is unnecessary but it may only be unnecessary for one of our targets (e.g. .net 10)... so you might be able to fix by making the Asn1 reference conditional on the tfm being net9 or earlier. This has worked for me in the past.

Let me know if this doesn't work and I guess we can use a pre-release of the testing package if there are no alternatives.